src/Kdyby/Replicator/Container.php

```diff
@@ -501,12 +532,12 @@ public function addComponent(Nette\ComponentModel\IComponent $component, $name,
 	public static function register($methodName = 'addDynamic')
 	{
 		if (self::$registered) {
-			Nette\Forms\Container::extensionMethod(self::$registered, function () {
+			Nette\Utils\ObjectMixin::setExtensionMethod('Nette\Forms\Container', self::$registered, function () {
```

Comment:
`Nette\Forms\Container::extensionMethod` is in fact more clean than `Nette\Utils\ObjectMixin::setExtensionMethod` because `extensionMethod` is now offical part of Container API.